src/20241108_Multi_OnboardAndEnableSUSDeLiquidEModeOnAaveV3MainnetAndLidoInstances/OnboardAndEnableSUSDeLiquidEModeOnAaveV3MainnetAndLidoInstances.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Onboard and Enable sUSDe liquid E-Mode on Aave v3 Mainnet and Lido Instances"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-enable-susde-liquid-e-mode-on-aave-v3-mainnet-and-lido-instance/1970"
+snapshot: "Direct-to-AIP"
+---
+
+## Simple Summary
+
+This proposal aims to enable sUSDe liquid E-Mode on Aave v3 Mainnet for the Main and Lido instances. By implementing this change, we seek to enhance capital efficiency for borrowers using sUSDe as collateral, particularly for borrowing other stablecoins and GHO. This is a Direct to AIP proposal.
```

Comment:
```suggestion
This proposal aims to enable sUSDe liquid E-Mode on Aave v3 Mainnet for the Main and Lido instances. By implementing this change, we seek to enhance capital efficiency for borrowers using sUSDe as collateral, particularly for borrowing other stablecoins. This is a Direct to AIP proposal.
```
GHO is not in the E-mode so it cannot be borrowed against sUSDe collateral, also as the LTV LT is so low